questionnaires/evening_assessment/evening_assessment_armt.json

```diff
@@ -0,0 +1,66 @@
+[
```

Comment:
same here the name should not be as ambiguous as `evening_assessment`. Perhaps adding `epi` to the name would help as these are specific to epilepsy.

Reply:
Good point.